docs/index.md

```diff
@@ -1,112 +0,0 @@
----
-title: Get Started
```

Comment:
why we delete index doc page?

Reply:
This document is outdated and we no longer link to it.